packages/plugin-infra/src/web3/useSocialAddressListAll.ts

```diff
@@ -26,18 +27,20 @@ export function useSocialAddressListAll(
 
     return useAsyncRetry(async () => {
         const userId = identity?.identifier?.userId
-        if (!userId || userId === '$unknown') return EMPTY_LIST
-
+        if (!userId || userId === '$unknown' || !identity?.publicKey) return EMPTY_LIST
         let cached = addressCache.get(userId)
+
         if (!cached) {
             cached = Promise.allSettled<AddressList>(
-                [EVM_IdentityService, SolanaIdentityService].map((x) => x?.lookup(identity) ?? []),
+                [EVM_IdentityService, SolanaIdentityService].map((x) => x?.lookup(identity, isOwnerIdentity) ?? []),
```

Comment:
Add `isOwner`, then you can do:

```suggestion
                [EVM_IdentityService, SolanaIdentityService].map((x) => x?.lookup(identity, identity.isOwner) ?? []),
```